cetav-ddw/lalibertad-cetav
# feat(add):"adding cards"

## Description

Adding 3 different types of cards 

### How to verify this PR



1. just need to check the structure on the main page after run dev
![Primeros cards](https://github.com/cetav-ddw/lalibertad-cetav/assets/106450347/5b60c6c4-f35e-412b-98b9-f6770d61f642)
![otro card](https://github.com/cetav-ddw/lalibertad-cetav/assets/106450347/44d7a849-b475-45cd-8e07-04f6ed28ba35)
